releasenotes/notes/logs-dd-url-path-prefix-53ce657fcf55a6e9.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Added support for a path prefix in the `logs_config.logs_dd_url` configuration.
```

```suggestion
    If a path is specified in the URL in `logs_config.logs_dd_url` use it as a prefix.
```

I think this is a bit more clear.